blog/2025/rippled-2.5.0.md

```diff
@@ -0,0 +1,178 @@
+---
+category: 2025
+date: "2025-06-04"
+template: '../../@theme/templates/blogpost'
+seo:
+    title: Introducing XRP Ledger version 2.5.0
+    description: rippled version 2.5.0 is now available. This version introduces new features and bug fixes.
+labels:
+    - rippled Release Notes
+markdown:
+    editPage:
+        hide: true
+---
+# Introducing XRP Ledger version 2.5.0
+
+Version 2.5.0 of `rippled`, the reference server implementation of the XRP Ledger protocol, is now available. This release adds new features and bug fixes, and introduces these amendments:
+
+- `TokenEscrow`
+- `Batch`
+- `PermissionedDEX`
+- `PermissionDelegation`
+- `AMMv1_3`
+- `EnforceNFTokenTrustlineV2`
+- `PayChanCancelAfter`
+
+
+## Action Required
+
+If you run an XRP Ledger server, upgrade to version 2.5.0 as soon as possible to ensure service continuity.
+
+Additionally, new amendments are now open for voting according to the XRP Ledger's [amendment process](../../docs/concepts/networks-and-servers/amendments.md), which enables protocol changes following two weeks of >80% support from trusted validators. The exact time that protocol changes take effect depends on the voting decisions of the decentralized network.
+
+
+## Install / Upgrade
+
+On supported platforms, see the [instructions on installing or updating `rippled`](../../docs/infrastructure/installation/index.md).
+
+| Package | SHA-256 |
+|:--------|:--------|
+| [RPM for Red Hat / CentOS (x86-64)](https://repos.ripple.com/repos/rippled-rpm/stable/rippled-2.5.0-1.el7.x86_64.rpm) | `7719be1889619a37a83795a9740d803bbc1c08b0bd8c755cbf266aeb68b875b6` |
+| [DEB for Ubuntu / Debian (x86-64)](https://repos.ripple.com/repos/rippled-deb/pool/stable/rippled_2.5.0-1_amd64.deb) | `d935f678624349e422dff1944a40acaf3e287b11244b4f5b5056cb343fc31e9d` |
+| [Portable Builds (Linux x86-64)](https://github.com/XRPLF/rippled-portable-builds) | (Use signature verification) |
+
+For other platforms, please [build from source](https://github.com/XRPLF/rippled/blob/master/BUILD.md). The most recent commit in the git log should be the change setting the version:
+
+```text
+commit 1e01cd34f7a216092ed779f291b43324c167167a
+Author: Michael Legleux <mlegleux@ripple.com>
+Date:   Tue Jun 17 10:38:07 2025 -0700
+
+    Set version to 2.5.0
+```
+
+{% admonition type="info" name="Note" %} After this release, we will no longer support GCC versions older than 12, Clang (including Apple Clang) versions older than 16, or Conan 1. {% /admonition %}
```

Comment:
Minor nitpick. Feels like this should be a warning instead of a note as we are stopping support after this release.

Reply:
Good idea, updating to warning.